content/en/docs/File Format/implementationstatus.md

```diff
@@ -113,12 +113,12 @@ Implementations:
 
 | Format                                       | C++   | Java  | Go    | Rust  | cuDF  |
 | -------------------------------------------- | ----- | ----- | ----- | ----- | ----- |
-| External column data (1)                     |       |       |       |       |  (W)  |
-| Row group "Sorting column" metadata (2)      |       |       |       |       |  (W)  |
-| Row group pruning using statistics           |       |       |       |       |  ✅   |
-| Row group pruning using bloom filter         |       |       |       |       |  ✅   |
-| Reading select columns only                  |       |       |       |       |  ✅   |
-| Page pruning using statistics                |       |       |       |       |  ❌   |
+| External column data (1)                     |  ❌   |  ✅   |       |       |  (W)  |
+| Row group "Sorting column" metadata (2)      |  ✅   |  ❌   |       |       |  (W)  |
+| Row group pruning using statistics           |  ✅   |  ✅   |       |       |  ✅   |
```

Comment:
Does C++ actually have an API for this? I don't remember.

Reply:
I think the dataset scanner supports this? Or it should not be included as parquet-cpp?

Reply:
Ah, yes, the dataset scanner does, but it's another component of Arrow C++, so I'm not sure it qualifies as a Parquet C++ feature.

Reply:
That makes sense since it is not in the `libparquet`. Let me mark it as unsupported.